src/clj/puppetlabs/services/jruby_pool_manager/jruby_schemas.clj

```diff
@@ -59,14 +59,18 @@
         values are `:jit`, `:force`, and `:off`.  Defaults to `:off`.
 
     * :max-active-instances - The maximum number of JRubyInstances that
-        will be pooled."
+        will be pooled.
+
+    * :environment-vars - A map of whitelisted environment variables and their values to be
+        whitelisted and visible to any Ruby code."
   {:ruby-load-path [schema/Str]
    :gem-home schema/Str
    :compile-mode SupportedJRubyCompileModes
    :borrow-timeout schema/Int
    :max-active-instances schema/Int
    :max-borrows-per-instance schema/Int
-   :lifecycle LifecycleFns})
+   :lifecycle LifecycleFns
+   :environment-vars {schema/Str schema/Str}})
```

Comment:
I would start out by just testing it manually - maybe put something in the `environment-vars` setting in the config file in puppet server and running it out of the repl and seeing if that works.